### DISCUSSION TITLE
Claim Overlap

### DISCUSSION THREAD

**Discussion by ImMalkah** [GriefPrevention/GriefPrevention#1900] [General]
Hello. I'd like to firstly say thank you for the great plugin, you're doing wonders.

I have an issue where every time I try to claim something, it says it overlaps an existing claim. However, when I pull out the stick to check what claim I'm overlapping, it says "No one has claimed this block". So I don't understand what I'm doing wrong but if someone can explain what's going on, that'd be great. 

By the way, to fix this I usually have to delete all the claims in the world and restart again, which obviously is not good in SMP where multiple people are playing.

**Comment by Jikoo** [comment]
Have you tried using the stick to visualize all nearby claims? A single click only examines the clicked point. If you're still having issues, please open a bug report including all of the requested information.

**Reply by ImMalkah** [reply]
I have spammed the stick to make sure certain blocks weren’t involved. So there is no obvious answer? If not I will create a bug report in that case.

**Reply by ImMalkah** [reply]
Unless there’s a way to visualize all nearby claims as you said? I only know about the right click method

**Reply by Jikoo** [reply]
Sneak and right click. That said, if you are receiving the conflict message, GP should be visualizing the conflicting area.

**Reply by ImMalkah** [reply]
Sneak right click is something I did not know! Turns out there was one block claimed accidentally and it was messing with the rest of the claim. Thank you for the info!
